collectd_plugin/collectd_plugin.py

```diff
@@ -227,9 +228,17 @@ def statsd_thread(
     if statsd_forward_prefix_hostname:
         statsd_forward_prefix = socket.gethostname()
         _log("Forwarding StatsD stats with hostname prefix %r", statsd_forward_prefix)
+
+    if data.get("monitor_endpoint") or (
+            data.get("carbon_host") and data.get("carbon_port")):
+        transport = "graphite_queue"
```

Comment:
@tipabu just a dump question, why we still init statsd server if we don't have `monitor_endpoint` and `carbon_host`?

Reply:
Also, is this (https://github.com/sivy/pystatsd) a correct repo that we use in this plugin?